### ISSUE TITLE
register error

### ISSUE THREAD

**Issue by PowerOlive** [minetest/minetest#8366]
##### Issue type

 - Bug report
 
##### Minetest version
5.0.0

```

##### OS / Hardware
<!-- General information about your hardware and operating system -->
Operating system:win10
CPU:i5intel

##### Summary
register output fail

##### Steps to reproduce
register

**Comment by PowerOlive** [comment]
![2019-03-12 20_52_53-Window](https://user-images.githubusercontent.com/20476787/54201984-fb05e300-4509-11e9-8f01-48c62ade99b2.png)

**Comment by PowerOlive** [comment]
hi

**Comment by srifqi** [comment]
Duplicate of #7948. Also, this bug has been fixed in #8258.
